astronomer/providers/google/cloud/example_dags/example_gcp_nuke.py

```diff
@@ -7,6 +8,7 @@
 from airflow.utils.timezone import datetime
 
 REGION = os.getenv("GCP_LOCATION", "us-central1")
+GCP_NUKE_DAG_SCHEDULE = os.getenv("GCP_NUKE_DAG_SCHEDULE", None)
```

Comment:
why did you choose the default value None and not `30 20 * * *`?

Reply:
Yes, we do the same in the AWS Nuke schedule. If we set the default schedule, since we now have 2 deployments, it will trigger the DAG from both deployments at the same time and it would be redundant(might also cause conflicts). We only schedule it from the CeleryExecutor deployment by setting this env var there, whereas the KubernetesExecutor deployment does not have this env var, so it won't be scheduled from there/.